frontend/src/components/GymKeyWith.tsx

```diff
@@ -38,6 +38,7 @@ const Container = styled.div`
 const ImageContainer = styled.div`
```

Comment:
https://github.com/rhdevs/rhapp/blob/6fa86e2770286d839c5f9078ac32598690537f86/frontend/src/components/GymKeyWith.tsx#L16

For this line it might be better to change it to `padding: 30px 0` since the spacing between the `GymStatus` and `KeyWIth` Component are a little close.
![2022-02-15 07 34 26 localhost 6b10566dea79](https://user-images.githubusercontent.com/46131429/153967422-23ee4ade-45ef-4558-a186-cc33f5da243a.png)

Reply:
You meant padding: 30px right, thanks for pointing out!

Reply:
c52221d5d27ae576679b8adeee965171545f0733

Reply:
`padding: 30px 0` just puts padding at top and bottom and nothing at left and right, although in this case it looks like just `padding: 30px` works as well